pages/test_analytics/test_splitting.md

```diff
@@ -0,0 +1,65 @@
+# Configuring test splitting
+
+Test splitting is the process of partitioning test suites to run in parallel across multiple Buildkite agents. Buildkite maintains the open source [test-splitter](https://github.com/buildkite/test-splitter) tool. This tool uses your Buildkite Test Analytics test suite data to intelligently partition tests throughout your test suites into multiple sets, such that each set of tests runs in parallel across your agents. This process is known as _orchestration_. Currently, the test-splitter tool only supports RSpec.
+
+## Dependencies
+
+Test splitting relies on execution timing data captured by the Buildkite test collectors to partition your tests evenly, so you will need to configure the [Ruby test collector](./ruby-collectors) for your test suite.
```

Comment:
Is the execution timing data obtained from previous builds?

Reply:
It is yeah